packages/frontend/src/components/common/Dashboard/DashboardHeader.tsx

```diff
@@ -528,6 +548,21 @@ const DashboardHeader = ({
                                     </Menu.Item>
                                 )}
 
+                                {userCanPromoteDashboard && (
+                                    <Menu.Item
```

Comment:
Do we need the tooltip "You must enable first an upstream project in settings > Data ops" ?